recipes/mss/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.2.1" %}
+
+package:
+    name: mss
+    version: {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: c08e72950a1150fd2a119a7fe766f966fe5a8b73c55fa02763c7a1e5a288f915
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+    skip: True  # [py3k]
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        # Relaxed versions of the run dependencies for conda-forge.
+        - python
+        - basemap-data-hires
```

Comment:
> currently I prefer it 1.0.7

That makes sense (and it is the latest release after all, conda-forge's version is a workaround to get the package working on Windows). Changing it now.

> I likly will be at euroscipy in Erlangen.

Not sure if @pelson attended that, but is he is you should look him up and ask about conda-forge and cartopy :wink: